bpf/lib/nodeport.h

```diff
@@ -901,10 +882,13 @@ static __always_inline int nodeport_lb6(struct __ctx_buff *ctx,
 skip_service_lookup:
 #ifdef ENABLE_NAT_46X64_GATEWAY
 		if (is_v4_in_v6_rfc8215((union v6addr *)&ip6->daddr)) {
+			ret = neigh_record_ip6(ctx);
+			if (ret < 0)
+				return ret;
```

Comment:
This is actually only needed for the stateful case, correct?
(ie. the `else` leg of the if-statement)